aspnetcore/client-side/import-export-interop.md

```diff
@@ -0,0 +1,348 @@
+| `Exception`                 | `Error`      |            | ✅                | ✅                    | |
+| `JSObject`                  | `Object`     |            | ✅                | ✅                    | ✅ |
+| `String`                    | `String`     |            | ✅                | ✅                    | ✅ |
+| `Object`                    | `Any`        |            | ✅                |                       | ✅ |
+| `Span<Byte>`                | `MemoryView` |            |                   |                        | |
+| `Span<Int32>`               | `MemoryView` |            |                   |                        | |
+| `Span<Double>`              | `MemoryView` |            |                   |                        | |
+| `ArraySegment<Byte>`        | `MemoryView` |            |                   |                        | |
+| `ArraySegment<Int32>`       | `MemoryView` |            |                   |                        | |
+| `ArraySegment<Double>`      | `MemoryView` |            |                   |                        | |
+| `Task`                      | `Promise`    |            |                   | ✅                    | |
+| `Action`                    | `Function`   |            |                   |                        | |
+| `Action<T1>`                | `Function`   |            |                   |                        | |
+| `Action<T1, T2>`            | `Function`   |            |                   |                        | |
+| `Action<T1, T2, T3>`        | `Function`   |            |                   |                        | |
+| `Func<TResult>`             | `Function`   |            |                   |                        | |
+| `Func<T1, TResult>`         | `Function`   |            |                   |                        | |
+| `Func<T1, T2, TResult>`     | `Function`   |            |                   |                        | |
+| `Func<T1, T2, T3, TResult>` | `Function`   |            |                   |                        | |
+
+The following conditions apply to type mapping and marshalled values:
+
+* The :::no-loc text="Array of"::: column indicates if the .NET type can be marshalled as a JS [`Array`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array). Example: C# `int[]` (`Int32`) mapped to JS `Array` of `Number`s.
+* When passing a JS value to C# with a value of the wrong type, the framework throws an exception in most cases. The framework doesn't perform compile-time type checking in JS.
+* `JSObject`, `Exception`, `Task` and `ArraySegment` create `GCHandle` and a proxy. You can trigger disposal in developer code or allow [.NET garbage collection (GC)](/dotnet/standard/garbage-collection/) to dispose of the objects later. These types carry significant performance overhead.
+* [`IMemoryView`](https://github.com/dotnet/runtime/blob/main/src/mono/wasm/runtime/marshal.ts)
```

Comment:
This is a link to the `IMemoryView` interface in TS, but the table above says `MemoryView`, which seems inconsistent.

Reply:
@pavelsavara Since `IMemoryView` isn't defined in dotnet.d.ts, should we document its structure here? It seems potentially problematic to refer to the   marshal.ts implementation.

Reply:
Agreed we should not link the source. Let not link anything until we fix the dotnet.d.ts

Reply:
I'll remove the link and go back to `MemoryView` until further notice.